src/main/fc/cli.c

```diff
@@ -124,10 +124,20 @@ extern uint8_t __config_end;
 
 
 static serialPort_t *cliPort;
+
+#ifdef STM32F1
+#define CLI_IN_BUFFER_SIZE 128
+#define CLI_OUT_BUFFER_SIZE 64
+#else
+// Space required to set array parameters
+#define CLI_IN_BUFFER_SIZE 256
+#define CLI_OUT_BUFFER_SIZE 256
```

Comment:
Yes - size is checked after each byte ... 

Whole bufWriter is quite non-systematic way to handle USB buffering